lib/ae_page_objects/element_proxy.rb

```diff
@@ -103,6 +103,14 @@ def method_missing(name, *args, &block)
       end
 
       implicit_element.__send__(name, *args, &block)
+    rescue AePageObjects::LoadingElementFailed
+      raise unless %w(to_s inspect).include? name.to_sym
```

Comment:
I included both `to_s` and `inspect` here because I figured that both methods would be better off returning a string rather than raising an error.  I could be convinced otherwise.